enum DrinkName {
  A_FEDORA_WITH_PERFUME_AND_A_PLUM
  BAD_TOUCH
  BEER
  BLEEDING_JANE
  BLOOM_LIGHT
  BLUE_FAIRY
  BRANDTINI
  COBALT_VELVET
  CREVICE_SPIKE
  FLAMING_MOAI
  FLUFFY_DREAM
  FRINGE_WEAVER
  FROTHY_WATER
  GRIZZLY_TEMPLE
  GUT_PUNCH
  MARSBLAST
  MERCURYBLAST
  MOONBLAST
  PIANO_MAN
  PIANO_WOMAN
  PILEDRIVER
  SPARKLE_STAR
  SUGAR_RUSH
  SUNSHINE_CLOUD
  SUPLEX
  ZEN_STAR
}

enum ScreenElementType {
  RESET
  LEFT_SLOT
  RIGHT_SLOT
  ADELHYDE
  BRONSON_EXTRACT
  POWDERED_DELTA
  FLANERGIDE
  KARMOTRINE
  BLENDER
  ADD_ICE
  AGE
  MIX
}

struct Coord {
  1: i32 x;
  2: i32 y;
}

struct ScreenElement {
  1: Coord centroid;
  2: i32 shortcut;
}
